### PR TITLE
Added CS questions

### PR DESCRIPTION
**Pull request by arpit-srivastava33**
Sorting questions

Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/general-cs-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,13 @@
 const generalCSQuiz = [
+   {
+    Question: "Which of the following sorting algorithms is stable in nature?",
+    Answer: "Merge Sort",
+    Distractor1: "Heap Sort",
+    Distractor2: "Quick Sort",
+    Distractor3: "Selection sort",
+    Explanation: "Merge sort is stable in nature.",
```
Comment (jdwilkin4) [inline src/data/general-cs-quiz.ts:8]:
```suggestion
    Explanation: "Stable sorting algorithms maintain the relative order of elements with equal values, or keys. Merge sort is stable in nature.",
```

File: src/data/general-cs-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,13 @@
...
+    Link: "https://www.freecodecamp.org/news/sorting-algorithms-explained-with-examples-in-python-java-and-c/#:~:text=Insertion%20sort%2C%20merge%20sort%2C%20and,and%20quick%20sort%20are%20unstable.",
```
Comment (jdwilkin4) [inline src/data/general-cs-quiz.ts:9]:
```suggestion
    Link: "https://www.freecodecamp.org/news/sorting-algorithms-explained-with-examples-in-python-java-and-c/#classification-of-a-sorting-algorithm",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by arpit-srivastava33** [comment]
I've applied those suggested changes.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**